tests/test_bucket_list.py

```diff
@@ -1,5 +1,5 @@
 import unittest
-from application.classes.bucket_list import BucketList
+from application.models.bucket_list import BucketList
 
 class BucketListTest(unittest.TestCase):
     def setUp(self):
```

Comment:
Whenever you define *setUp*, it's highly encouraged to define *tearDown* in order to clear what you set up